build/ci/clean_demo_pr.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -xe
+set +o pipefail
+
+export failed_k8s_checks=false
+export failed_openshift_checks=false
+
+export repo_pr=`gh pr list --repo $github_csiblock_community_operators_repository | grep $community_operators_kubernetes_branch`
+if [[ "$repo_pr" == *"$community_operators_kubernetes_branch"* ]]; then
+  sleep 5
```

Comment:
Why do we need to sleep 5 seconds?

Reply:
We sleep for 10 seconds the first time.
Does it make sense?

Reply:
I put 5 seconds at the beginning just for insurance that the checks are running